modules/ROOT/pages/managing-queues.adoc

```diff
@@ -9,22 +9,29 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
-CloudHub includes a *Queues* tab in the left navigation menu where you can view and control any Virtual Machine (VM) or Staged Event-Driven Architecture (http://en.wikipedia.org/wiki/Staged_event-driven_architecture[SEDA]) queues within the flows of your deployed applications:
+You can view and control any Virtual Machine (VM) or Staged Event-Driven Architecture (http://en.wikipedia.org/wiki/Staged_event-driven_architecture[SEDA]) queues within the flows of your deployed applications from the *Queues* tab in CloudHub.
 
-[NOTE]
-Queues are a feature that is currently only available for applications deployed to CloudHub workers. They are not available for applications that you deploy via the Runtime Manager to local servers. See xref:deployment-strategies.adoc[Deployment Strategies] for more details.
+Queues are currently only available for applications deployed to CloudHub workers. They are not available for applications that you deploy with the Runtime Manager to local servers. See xref:deployment-strategies.adoc[Deployment Strategies] for more details.
 
 image::queuestab.png[QueuesTab]
 
-CloudHub automatically detects the queues from within your flow and lists them on the Queues tab. You can see the number of queued and in-flight messages on each queue, view a snapshot of the queue throughput for the past 24 hours, or clear all pending messages on a queue – all during runtime, without making any changes in the underlying application. You can also enable data-at-rest encryption for your persistent queues, this ensures that any shared application data written out to a persistent queue is encrypted.
+CloudHub automatically detects the queues from within your flow and lists them on the *Queues* tab.
+
+Without making any changes in the underlying application, you can:
+
+* See the number of queued and in-flight messages on each queue.
+* View a snapshot of the queue throughput for the past 24 hours.
+* Clear all pending messages on a queue.
+
+You can also enable data-at-rest encryption for your persistent queues to ensure that any shared application data written to a persistent queue is encrypted.
 
 == Use Cases
 
-The *Queues* tab is especially useful for the following use cases:
+The *Queues* tab is useful for the following use cases:
 
 * You want runtime visibility into the queues in your application.
 * You are testing your application to observe whether messages are traveling through your queues as expected.
-* Your application is stuck processing a particular message and you want to flush the messages from just that queue to resolve the issue.
+* Your application is stuck processing a message and you want to flush the messages from just that queue to resolve the issue.
```

Comment:
I might say "...flush the messages from only that queue..."
"Just" is probably on the list of ambiguous words.

Reply:
Yep.